### PR TITLE
SubQuery Featured

### PR DESCRIPTION
**Pull request by bgodlin**
## Purpose

Hello Concordium team,

We are submitting this contribution with the intention of incorporating SubQuery into your project documentation. As you can observe, we have already prepared the initial project. Kindly inform us if any adjustments are necessary on our part.

## Changes

We established a distinct category `Indexers` because of their crucial role in querying historical data. We incorporated our indexer by referring to pre-existing quickstart guides and starter projects.

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: source/mainnet/net/indexers/intro.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+========
+Overview
+========
+
+Indexers, in a broad context, play a fundamental role in organising and optimising data retrieval within various systems. These tools act as navigational aids, allowing efficient access to specific information by creating structured indexes. In the realm of databases and information management, indexers enhance query performance by creating a roadmap to swiftly locate data entries.
```
Comment (dg-concordium) [inline source/mainnet/net/indexers/intro.rst:5]:
```suggestion
Indexers, in a broad context, play a fundamental role in organizing and optimizing data retrieval within various systems. These tools act as navigational aids, allowing efficient access to specific information by creating structured indexes. In the realm of databases and information management, indexers enhance query performance by creating a roadmap to swiftly locate data entries. 
```

File: source/mainnet/net/indexers/intro.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
+========
+Overview
+========
+
+Indexers, in a broad context, play a fundamental role in organising and optimising data retrieval within various systems. These tools act as navigational aids, allowing efficient access to specific information by creating structured indexes. In the realm of databases and information management, indexers enhance query performance by creating a roadmap to swiftly locate data entries. 
+
+In the context of blockchain and dApps, indexers go beyond traditional databases, facilitating streamlined access to on-chain data. This includes transaction histories, smart contract states, and event logs. In the dynamic and decentralised world of blockchain, indexers contribute to the efficiency of data queries, supporting real-time updates and ensuring the seamless functionality of diverse applications and platforms.
```
Comment (dg-concordium) [inline source/mainnet/net/indexers/intro.rst:7]:
```suggestion
In the context of blockchain and dApps, indexers go beyond traditional databases, facilitating streamlined access to on-chain data. This includes transaction histories, smart contract states, and event logs. In the dynamic and decentralized world of blockchain, indexers contribute to the efficiency of data queries, supporting real-time updates and ensuring the seamless functionality of diverse applications and platforms.
```

File: source/mainnet/net/indexers/intro.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+There are several indexer solutions available, each offering different levels of decentralisation, ease of development, and performance for you to consider. These solutions serve as intermediaries to assist in indexing the Concordium network.
```
Comment (dg-concordium) [inline source/mainnet/net/indexers/intro.rst:9]:
```suggestion
There are several indexer solutions available, each offering different levels of decentralization, ease of development, and performance for you to consider. These solutions serve as intermediaries to assist in indexing the Concordium network.
```

File: source/mainnet/net/indexers/subquery.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
+========
+SubQuery
+========
+
+SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Concordium, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.
```
Comment (dg-concordium) [inline source/mainnet/net/indexers/subquery.rst:5]:
```suggestion
SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open-source, and decentralized APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralized applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Concordium, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.
```

File: source/mainnet/net/indexers/subquery.rst [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Another one of SubQuery's competitive advantages is the ability to aggregate data not only within a chain but across multiple blockchains all within a single project. This allows the creation of feature-rich dashboard analytics and multi-chain block scanners.
```
Comment (dg-concordium) [inline source/mainnet/net/indexers/subquery.rst:7]:
```suggestion
Another of SubQuery's competitive advantages is the ability to aggregate data not only within a chain but across multiple blockchains all within a single project. This allows the creation of feature-rich dashboard analytics and multi-chain block scanners.
```

File: source/mainnet/net/indexers/subquery.rst [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+Other advantages include superior performance with multiple RPC endpoint configurations, multi-worker capabilities and a configurable caching architecture. To find out more, visit our [documentation](https://academy.subquery/).
```
Comment (dg-concordium) [inline source/mainnet/net/indexers/subquery.rst:9]:
```suggestion
Other advantages include superior performance with multiple RPC endpoint configurations, multi-worker capabilities, and a configurable caching architecture. To find out more, visit SubQuery's `documentation <https://academy.subquery/>`_.
```

File: source/mainnet/net/indexers/subquery.rst [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+- SubQuery Docs: [SubQuery Academy (Documentation)](https://academy.subquery.network/)
+- Intro Quick Start Guide: [1. Create a New Project](https://academy.subquery.network/quickstart/quickstart.html)
+- [Concordium Quick Start Guide](https://academy.subquery.network/quickstart/quickstart_chains/concordium.html)
+
+Getting Started
+---------------------
+
+Take a look at this SubQuery Starter Project that introduces SubQuery's Concordium support by indexing [Concordium](https://github.com/subquery/concordium-subql-starter/tree/main/Concordium/concordium-testnet-starter).
+
+You can also follow along this [step by step guide](https://academy.subquery.network/quickstart/quickstart.html) to get familiar with SubQuery.
```
Comment (dg-concordium) [inline source/mainnet/net/indexers/subquery.rst:21]:
```suggestion
You can also follow this [step-by-step guide](https://academy.subquery.network/quickstart/quickstart.html) to get familiar with SubQuery.
```

File: source/mainnet/net/indexers/subquery.rst [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+Running and Hosting your Concordium SubQuery APIs
+---------------------
```
Comment (dg-concordium) [inline source/mainnet/net/indexers/subquery.rst:24]:
```suggestion
-----------------------------------------------------------------------
```

File: source/mainnet/net/indexers/subquery.rst [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+SubQuery is open-source, meaning you have the freedom to run it in the following three ways:
+- Locally on your own computer (or a cloud provider of your choosing), [view the instructions on how to run SubQuery Locally](https://academy.subquery.network/run_publish/run.html).
+- You can publish it to SubQuery's enterprise-level [Managed Service](https://managedservice.subquery.network/), where we'll host your SubQuery project in production ready services for mission critical data with zero-downtime blue/green deployments. There even is a generous free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).
```
Comment (dg-concordium) [inline source/mainnet/net/indexers/subquery.rst:28]:
```suggestion
- You can publish it to SubQuery's enterprise-level [Managed Service](https://managedservice.subquery.network/), where we'll host your SubQuery project in production ready services for mission critical data with zero-downtime blue/green deployments. There is even a generous free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).
```

File: source/mainnet/net/indexers/subquery.rst [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+- You can publish it to the decentralised [SubQuery Network](https://subquery.network/network), the most open, performant, reliable, and scalable data service for dApp developers. The SubQuery Network indexes and services data to the global community in an incentivised and verifiable way and supports Concordium from launch.
```
Comment (dg-concordium) [inline source/mainnet/net/indexers/subquery.rst:29]:
```suggestion
- You can publish it to the decentralized [SubQuery Network](https://subquery.network/network), the most open, performant, reliable, and scalable data service for dApp developers. The SubQuery Network indexes and services data to the global community in an incentivized and verifiable way and supports Concordium from launch.
```

**Review by dg-concordium — CHANGES_REQUESTED** [review]
I think I forgot to mention that we use US English. Sorry about that.

File: source/mainnet/net/indexers/subquery.rst [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+Take a look at this SubQuery Starter Project that introduces SubQuery's Concordium support by indexing [Concordium](https://github.com/subquery/concordium-subql-starter/tree/main/Concordium/concordium-testnet-starter).
+
+You can also follow this [step-by-step guide](https://academy.subquery.network/quickstart/quickstart.html) to get familiar with SubQuery.
+
+Running and Hosting your Concordium SubQuery APIs
+-----------------------------------------------------------------------
+
+SubQuery is open-source, meaning you have the freedom to run it in the following three ways:
+- Locally on your own computer (or a cloud provider of your choosing), [view the instructions on how to run SubQuery Locally](https://academy.subquery.network/run_publish/run.html).
```
Comment (abizjak) [inline source/mainnet/net/indexers/subquery.rst:27]: Please use RST link syntax.
```suggestion
- Locally on your own computer (or a cloud provider of your choosing), `view the instructions on how to run SubQuery Locally <https://academy.subquery.network/run_publish/run.html>`_.
```

**Review by dg-concordium — APPROVED** [review]

**Review by abizjak — APPROVED** [review]

**Pull request merged**